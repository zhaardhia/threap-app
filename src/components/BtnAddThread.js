import React from 'react';
import PropTypes from 'prop-types';

const BtnAddThread = ({ toggleModal }) => {
  const x = 0;
  return (
    <button className=" bg-white fixed right-[30rem] bottom-[38px] px-5 py-3 rounded-full shadow-lg" onClick={() => toggleModal(true)}>
      <p className="text-black text-lg">+</p>
    </button>
  );
};

BtnAddThread.propTypes = {
  toggleModal: PropTypes.func.isRequired,
};

export default BtnAddThread;
