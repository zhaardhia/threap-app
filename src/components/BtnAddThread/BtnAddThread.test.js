/**
 * test scenario for Footer
 *
 * - Footer component
 *  - should render Footer component successfully
 *
 */
import { render } from '@testing-library/react';
import BtnAddThread from '../BtnAddThread';
import '@testing-library/jest-dom';

describe('BtnAddThread component', () => {
  it('should render BtnAddThread component successfully', async () => {
    const { container } = render(<BtnAddThread toggleModal={() => {}} />);
    expect(container).toMatchSnapshot();
  });
});
