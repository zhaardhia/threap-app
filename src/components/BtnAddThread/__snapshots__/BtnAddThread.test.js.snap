// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BtnAddThread component should render BtnAddThread component successfully 1`] = `
<div>
  <button
    class=" bg-white fixed right-[16%] bottom-[38px] px-5 py-3 rounded-full shadow-lg"
  >
    <p
      class="text-black text-lg"
    >
      +
    </p>
  </button>
</div>
`;
